examples/get_sensor_history.py

```diff
@@ -151,6 +158,11 @@ def main():
         dest='verbose', action="store_true",
         default=False,
         help="provide extremely verbose output.")
+    parser.add_argument(
+        '--additional-fields',
+        default='status',
+        help="Additional fields to be queried from katstore database")
```

Comment:
Please add the available "additional fields" to the help string such that a user can know how these are named e.g. "status,value_time"

Reply:
I don't think this should even be a command line argument - see comments in the `sensor_history` method.